### PR TITLE
add missing `<meta charset="UTF-8">` in `head` of All `contexts` and `tests`

### LINKED ISSUE #156: [Bug] Missing `<meta charset="UTF-8">` in Head of All contexts and tests
## Description
The `<meta charset="UTF-8">` tag is missing in the `<head>` section of all context and tests files. This tag is essential for proper character encoding and should be included in all HTML documents.

## Reproduction
1. Open any context HTML file (e.g., [program-page.html]
2. Inspect the` <head> ` section.
3. Notice the absence of the `<meta charset="UTF-8">`  tag.

## Expectation
The `<meta charset="UTF-8">` tag should be present in the <head> section of all context HTML files.
## Screenshots
meta-issue

## Environment

 - Device: (_eg._ iPhone Xs; laptop)
 - OS: (_eg._ iOS 13.5; Fedora 32)
 - Browser: (_eg._ Safari; Firefox)
 - Version: (_eg._ 13; 73)
 - Other info: (_eg._ display resolution, ease-of-access settings)

## Additional context
 This issue affects all context HTML files and should be addressed globally.

## Resolution
- [x] I would be interested in resolving this bug.

### PR DESCRIPTION
**Pull request by Daodu111**
## Fixes

- Fixes #156 by @possumbilities 

## Description

I have added the `<meta charset="UTF-8">` to all the HTML files.

## Checklist

- [ ] My pull request has a descriptive title (not a vague title like `Update
  index.md`).
- [ ] My pull request targets the *default* branch of the repository (`main` or `master`).
- [ ] My commit messages follow [best practices][best_practices].
- [ ] My code follows the established code style of the repository.
- [ ] I added or updated tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [ ] I tried running the project locally and verified that there are no
  visible errors.

[best_practices]:https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53

## Developer Certificate of Origin

For the purposes of this DCO, "license" is equivalent to "license or public domain dedication," and "open source license" is equivalent to "open content license or public domain dedication."
<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5502
+}
```
Comment (possumbilities) [inline .vscode/settings.json]: this change is unrelated to this PR

Comment (Daodu111) [reply .vscode/settings.json]: I had to put the .vscode file in the git ignore file because in a PR I submitted earlier it was requested for another issue. Also it was in this PR that I added it. 

Please can you explain a way for me to solve that and also did I meet all the expectations?

Thank you

Comment (possumbilities) [reply .vscode/settings.json]: When you add changes, you should add only the files related to the work you are doing rather than `git add -A`, in this way it makes sure you can have files present that are not added to a commit. 

You would need to remove this file, commit that, and then if you want to add it back be sure to not re add it to the commit again.

**Comment by possumbilities** [comment]
additionally, the work should include all the `html` files within the `specimen` folder, not just the `contexts` (`tests` too)

File: .vscode/settings.json (hunk 1 above)
Comment (Daodu111) [reply .vscode/settings.json]: I have removed the file and committed it, also the changes is in all the HTML files including the tests.

**Comment by possumbilities** [comment]
Fixed by: 
- #217

Closing.
